pkg/pkg/add/add.go

```diff
@@ -23,6 +24,10 @@ func Run(pkgManifestFilename string, templateName, outputFolder string) (*AddRes
 
 	latestReleases, err := githubreleases.GetLatestReleases()
 	if err != nil {
+		if strings.Contains(err.Error(), "secret not found in keyring") {
+			fmt.Println(githubreleases.HelpMessage)
```

Comment:
I think we should print messages like these to `os.Stderr` instead so we don't pollute the standard output of the app.

And at the same time we can change to formatted output so we don't need duplicate `fmt.Println`

```suggestion
			fmt.Fprintf(os.Stderr, "%s\n\n", githubreleases.HelpMessage)
```